prow/inrepconfig.md

```diff
@@ -0,0 +1,66 @@
+# Inrepoconfig
+
+Inrepoconfig is a Prow feature that allows versioning Presubmit jobs in the same repository
+that also holds the code. If enabled, Prow will use both the centrally-defined jobs and the
+ones defined in the code repository. The latter ones are dynamically loaded on-demand.
+
+To enable it, add the following to your Prows `config.yaml`:
+
+```
+in_repo_config:
+  enabled:
+    # The key can be one of "*" for "globally", "org" for the GitHub organization or "org/repo" for
+    # a specific repo. The narrowest match is used.
+    "kubernetes/kubernetes": true
+```
+
+Afterwards, you need to add a config verification job to make sure people people get told about
```

Comment:
Do we want to ship like this? Why not just dynamically generate this presubmit?

Reply:
Well, primarily this documents the current state of affairs. This doesn't mean we can not improve it, but thats how it is today.